src/Lumi/Components/Form/Table.purs

```diff
@@ -103,10 +126,19 @@ editableTable { addLabel, defaultValue, formBuilder: builder, maxRows, summary }
                   , children: [ summary ]
                   }
             , rows: Left $ mapWithIndex Tuple rows
-            , onRowAdd: foldMap (onChange <<< flip Array.snoc) defaultValue
+            , onRowAdd:
+                for_ addRow \addRow' -> launchAff_ do
+                  row <- addRow'
+                  liftEffect $ onChange (flip Array.snoc row)
```

Comment:
If this needs to be async can we capture the loading state here so there's UI feedback when this is slow?

Reply:
We can, yes, but async may also mean something like: open a modal window with another form that lets the user choose something, in which case do we also want that same UI feedback?

Reply:
True, and that seems the more likely use case so maybe it's ok for now